pkgs/applications/science/math/mpsolve/default.nix

```diff
@@ -1,25 +1,34 @@
 { lib
 , stdenv
-, fetchurl
+, fetchFromGitHub
+, autoreconfHook
+, flex
+, gitUpdater
 , gmp
+, gtk3
 , pkg-config
 , qtbase
 , wrapQtAppsHook
-, gtk3
+, yacc
```

Comment:
yacc has been renamed to bison

Reply:
Thanks. I've updated the commit accordingly.